assets/model_monitoring/components/data_drift/data_drift_compute_metrics/spec.yaml

```diff
@@ -65,6 +65,7 @@ conf:
       - pip:
         - azure-storage-file-datalake~=12.8.0
         - azure-ai-ml~=1.13.0
+        - azureml-mlflow~=1.49.0
```

Comment:
momo can't work with latest protobuf, so pin mlflow, thus pin the dependent protobuf to its old version, is that the fix?

Reply:
Actually I didn't figure out the protobuf version, from the call stack it's caused by mlflow. I tested in spark session seems I can fix it with this pin.